{{cookiecutter.python_name}}/.github/workflows/build.yml

```diff
@@ -54,18 +76,97 @@ jobs:
     - name: Install Python
       uses: actions/setup-python@v2
       with:
-        python-version: '3.8'
+        # Ensure the python version matches the one in build
+        python-version: '3.9'
         architecture: 'x64'
     - uses: actions/download-artifact@v2
       with:
-        name: myextension-sdist
+        name: extension-artifacts
     - name: Install and Test
       run: |
         set -eux
         # Remove NodeJS, twice to take care of system and locally installed node versions.
         sudo rm -rf $(which node)
         sudo rm -rf $(which node)
-        pip install myextension.tar.gz
+        pip install ./{{ cookiecutter.python_name }}*.tar.gz
         pip install jupyterlab
+{% if cookiecutter.kind.lower() == 'server' %}
+        jupyter server extension list
+        jupyter server extension list 2>&1 | grep -ie "{{ cookiecutter.python_name }}.*OK"
+{% endif %}
+        jupyter labextension list
         jupyter labextension list 2>&1 | grep -ie "{{ cookiecutter.labextension_name }}.*OK"
         python -m jupyterlab.browser_check --no-chrome-test
+
+  integration-tests:
+    name: Integration tests
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Ensure the python version matches the one in build
```

Comment:
actually _doesn't_ need to match, per se, and gives an opportunity to test multiple pythons.

Reply:
True I remove that additional constraint.